todo_app/data/view_model.py

```diff
@@ -16,27 +16,27 @@ def items(self):
 
     @property
     def todo_items(self):
-        return list(filter(lambda item: item.status == "To Do", self._items))
```

Comment:
I would probably argue that this is a code smell - why should the ViewModel care that our data persistence has changed? The outside behaviour isn't changing, so there's definitely a case to be made that this suggests that our names are too tightly tied to Trello/Mongo here, since the `item` class is precisely meant to protect the rest of the codebase from caring about those changes